src/pages/search/PublicationYearIntervalFilter.tsx

```diff
@@ -65,7 +65,7 @@ export const PublicationYearIntervalFilter = ({ datePickerProps, boxProps }: Pub
         {...datePickerProps}
         label={t('search.year_to')}
         value={selectedYearBeforeDate}
```

Comment:
Kunne sikkert oppnådd samme med mindre endinger ved å bare sikre at `value` aldri er `undefined` (siden `minDate` og `maxDate` ikke kan være `null` igjen)
```suggestion
        value={selectedYearBeforeDate ?? null}
```